packages/rocket-fileuploader-aws-infrastructure/package.json

```
@@ -0,0 +1,56 @@
+{
+  "name": "@boostercloud/rocket-fileuploader-aws-infrastructure",
+  "version": "0.9.5699",
+  "description": "Booster rocket to upload files in File Storage. A new event will be stored in the Events Store as a consequence.",
+  "keywords": [
+    "Booster",
+    "plugin",
+    "aws",
+    "s3-trigger-lambda"
+  ],
+  "author": "Booster Cloud",
+  "homepage": "https://booster.cloud",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/boostercloud/booster.git"
+  },
+  "dependencies": {
+    "@aws-cdk/aws-dynamodb": "1.67.0",
+    "@aws-cdk/aws-lambda": "1.67.0",
+    "@aws-cdk/aws-lambda-event-sources": "1.67.0",
+    "@boostercloud/framework-types": "^0.11.1",
```

I think we can also move this one to devDependencies:
```suggestion
    "@boostercloud/framework-types": "0.11.2",
```